Foundation/MeasurementFormatter.swift

```diff
@@ -77,7 +77,3 @@ open class MeasurementFormatter : Formatter, NSSecureCoding {
     open override func encode(with aCoder: NSCoder) { NSUnimplemented() }
     public static var supportsSecureCoding: Bool { return true }
 }
-
-extension MeasurementFormatter {
-    public func string<UnitType>(from measurement: Measurement<UnitType>) -> String { NSUnimplemented() }
-}
```

Comment:
Why did this function go away?

Reply:
Since that is implemented here: https://github.com/apple/swift-corelibs-foundation/pull/1353/files#diff-4454b507f83169b4745628b0daeb7207R257

Reply:
Totally missed that. Thanks, this looks great.